### ISSUE TITLE
Performance command for telegram

### ISSUE THREAD

**Issue by agarzon** [CoinAlpha/hummingbot#2175]
**Feature Suggestion**
To add a new command for Telegram to focus in the summary of the current bot situation, means:  _Inventory, Markets and Performance_. This is already shown in the **history** command. But histrory should focus only in the recent transacionts and let the _Inventory, Markets and Performance_. to be in a separate command.

**Impact**
Curently the only way I can see how my bot is behaving from telegram is running the command **history**, which throws the last 100 transactions first, which is useless information for what I'm looking for,

**Additional context**
So, basically you can remove _Inventory, Markets and Performance_ from history and show it in a new command.
